stock_bin_location/README.rst

```diff
@@ -0,0 +1,91 @@
+==================================
+Documents for Lot/Tracking Numbers
+==================================
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-LGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+    :alt: License: LGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-ursais%2Fosi--addons-lightgray.png?logo=github
+    :target: https://github.com/ursais/osi-addons/tree/14.0/documents_stock_production_lot
+    :alt: ursais/osi-addons
+
+|badge1| |badge2| |badge3|
+
+This module allows you to introduce a bin location value in the Transfers, and read it in Lots/serial numbers and Inventory Report
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+To use this module, you need to:
+
+Purchase App:
+
+Click on 'Purchase' menu and then click on 'Requests for Quotation'.
```

Comment:
I don't think the purchase app info here is needed as this module doesn't depend on the purchase. So, only the inventory section is needed.

Reply:
I think it's done. I've changed the year to 2022, edited the Usage in the README file and uploaded only this module modifications. Thanks for sharing the squashing article.